### PR TITLE
[Merged by Bors] - feat: add channel action runtime handler (PL-38)

### PR DESCRIPTION
**Pull request by pmvrmc**
**Fixes or implements PL-38**

### Brief description. What is this change?
Adds a new runtime handlers that handles CHANNEL_ACTION nodes. It will extract variables and passdown the payload to the adapter

Since this is just adding the runtime handler, we are good to merge when ready since nothing will consume it

### CODE REVIEW

**Review by DecathectZero — APPROVED** [review]

File: lib/services/runtime/handlers/channelAction.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+import { findEventMatcher } from './event';
+
+const utilsObj = {
+  commandHandler: CommandHandler(),
+  findEventMatcher,
+};
+
+export const ChannelActionHandler: HandlerFactory<BaseNode.ChannelAction.Node, typeof utilsObj> = (utils) => ({
+  canHandle: (node) => node.type === BaseNode.NodeType.CHANNEL_ACTION,
+  handle: (node, runtime, variables) => {
```
Comment (DecathectZero) [inline lib/services/runtime/handlers/channelAction.ts:18]: could we just call `_V1Handler.handle()` from this handle? We would need to modify the `node` so it conforms with the _v1 format but it avoids duplicating code like this

Comment (pmvrmc) [reply lib/services/runtime/handlers/channelAction.ts]: reafactored it, do you agree with it?

**Review by DecathectZero — APPROVED** [review]

File: lib/services/runtime/handlers/channelAction.ts (hunk 1 above)
Comment (DecathectZero) [reply lib/services/runtime/handlers/channelAction.ts]: yup! Sorry about the late reply - looks good

**Pull request closed without merging**